src/atldld/plot.py

```diff
@@ -0,0 +1,123 @@
+        the corner in the axes origin. The second array axis contains the 3D
+        coordinates of the corners in the standard PIR references space.
+
+    plane_of_section
+        The plane of section of the dataset. Can be either coronal or sagittal.
+
+    Returns
+    -------
+    fig
+        The figure with the plot.
+    """
+    ref_space_size = np.array(REF_DIM_25UM)
+    p, i, r = 0, 1, 2
+    labels = {
+        p: "p (coronal)",
+        i: "i (transversal)",
+        r: "r (sagittal)",
+    }
+    # We'll plot the views of all four edges in counterclockwise order starting
+    # with the bottom edge. The last two x-axes are inverted so that the edge
+    # vertices on the right of a plot appear on the left of the following plot.
+    edges = [[0, 1], [1, 2], [2, 3], [3, 0]]
+    titles = ["Bottom Edges", "Right Edges", "Top Edges", "Left Edges"]
+    inverts = [False, False, True, True]
+
+    # Depending on the plane of section the views are different
+    if plane_of_section == PlaneOfSection.CORONAL:
+        y_axis = p
+        x_axes = [r, i, r, i]
+    elif plane_of_section == PlaneOfSection.SAGITTAL:
+        y_axis = r
+        x_axes = [p, i, p, i]
+    else:  # pragma: no cover
+        raise NotImplementedError(f"Unknown plane of section: {plane_of_section}")
+
+    # The figure width is fixed and arbitrary. (Is there a more clever choice?)
+    # The figure height is computed to match the ratio between the total width
+    # and height of all subplots. This way the ratios of the x and y axes are
+    # roughly the same (but not quite since the in-between spaces, titles, etc.
+    # are not taken into account...)
+    plot_width = sum(ref_space_size[x_axis] for x_axis in x_axes)
+    plot_height = ref_space_size[y_axis]
+    fig_width_inches = 14
+    fig_height_inches = fig_width_inches * plot_height / plot_width
+
+    fig = Figure(figsize=(fig_width_inches, fig_height_inches))
+    fig.set_tight_layout(True)
+    # The width ratios of subplots are based on the reference volume dimensions,
+    # this way the scales of the x-axes roughly match.
+    axs = fig.subplots(
+        ncols=4,
+        sharey=True,
+        gridspec_kw={"width_ratios": [ref_space_size[x_axis] for x_axis in x_axes]},
+    )
+    # Y-label only on the left-most plot because it's the same for all plots
+    axs[0].set_ylabel(labels[y_axis], fontsize=16)
+
+    # The actual plotting
+    for ax, edge, x_axis, invert, title in zip(axs, edges, x_axes, inverts, titles):
+        ax.grid(True, linestyle=":", color="gray")
+        ax.set_ylim((0, ref_space_size[y_axis]))
+        ax.set_title(title)
+        ax.set_xlabel(labels[x_axis], fontsize=16)
+        ax.axvline(0, color="blue", linestyle=":")
+        ax.axvline(ref_space_size[x_axis], color="blue", linestyle=":")
+        for corners in all_corners:
+            points = corners[np.ix_(edge, [x_axis, y_axis])]
+            coords = points.T / 25
```

Comment:
See my previous comment https://github.com/BlueBrain/Atlas-Download-Tools/pull/66#discussion_r698296495. 

In this specific case you are right that it would just change the ticks. However, I am not sure if an external person would right away understand that this 25 in the code stands for microns. IMO we clearly declare/document that this command assumes `downasmple_ref=25` or we turn it into a CLI parameter.

Reply:
I agree, the bare 25 might look a bit cryptic. I made some adjustments in 7ca3c02, what do you think?

Reply:
It's perfect! Thank you!